### ISSUE TITLE
Mounting instruction for V6-R6 Hotend

### ISSUE THREAD

**Issue by TheMaxx2804** [sorted01/Filametrix#44]
Hi there,

i am missing the mounting / building instruction for the V6_R6 Hotend.
As shown in the picture the hotend is standing out some mm from the print head back and front piece.
In this case i do not know how the knife would be mounted to work correctly.

Thank you in advance for any suggestions.

![Screenshot 2024-01-10 172639](https://github.com/sorted01/Filametrix/assets/67709265/5c34b4b4-c150-47e3-9d03-9c65e4743e7d)

**Comment by mole6999** [comment]
I could use help woth the same problem. Has anyone figures it out yet?

**Comment by protorayn** [comment]
Yes. remove the black plastic ptfe  adapter. I removed it and printed a sleeve that holds the ptfe in place for cutting.

**Comment by ThunderClap86** [comment]
> > printed a sleeve that holds the ptfe in place for cutting.
> 
> So the cutting knife has to be cut in length that it does stay on the silver collar and that it could not flip down that collar.
Good suggestions on both of these. 

Have a STL or CAD item for the collar you made to replace the pressfit collar in the pic above?

**Comment by ThunderClap86** [comment]
> Yes. remove the black plastic ptfe adapter. I removed it and printed a sleeve that holds the ptfe in place for cutting.

Guess it's you I need to be asking for the file(s) if you are willing to share.

**Comment by protorayn** [comment]
I'll put it up on printables when I get home tonight

**Comment by protorayn** [comment]
> > Yes. remove the black plastic ptfe adapter. I removed it and printed a sleeve that holds the ptfe in place for cutting.
> 
> Guess it's you I need to be asking for the file(s) if you are willing to share.

https://www.printables.com/model/914674-filament-guide-sleeve-for-dragonfly-bmo

This is the sleeve I made for replacing the bowden collet.

**Comment by alfadex** [comment]
@TheMaxx2804  Says the problem is here as you can see in the photo. Metal hotend is 1 mm above printed part and the cutter stop to the metal. Has somebody find any solution?

![Image](https://github.com/user-attachments/assets/24e8be31-327f-496a-8289-d09ed78a90f4)
